spec/helpers/node-setup.js

```diff
@@ -1,4 +1,7 @@
-// Setup for running Mocha via Node
+// Setup for running Mocha via
+
+require( "babel-polyfill" );
+
 var jsdom = require( "jsdom" ).jsdom;
 global.document = jsdom( "<html><body></body></html>" );
 global.window = document.parentWindow;
```

Comment:
If you change this line to `global.window = document.defaultView;` it should get your mocha tests running correcrtly